nservicebus/operations/opentelemetry_host-identifier_core_[8,).partial.md

```diff
@@ -22,7 +22,7 @@ Guid serviceInstanceId = GuidUtility.Create(new Guid("4d63009a-8d0f-11ee-aad7-4c
 
 // OpenTelemetry
 services.AddOpenTelemetry()
-    .ConfigureResource(rb => rb.AddService("MyService", serviceInstanceId: serviceInstanceId.ToString()))
+    .ConfigureResource(builder => builder.AddService("MyService", serviceInstanceId: serviceInstanceId.ToString()))
```

Comment:
We may want to address David's comment on this part to make this a snippet

Reply:
Fixed.